l10n_ec_ote/models/l10n_ec_parish.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class L10nEcParish(models.Model):
+    _name = "l10n.ec.parish"
```

Comment:
I think you should continue with this module:

https://github.com/OCA/partner-contact/pull/1231

Reply:
issue was closed, @pedrobaeza you do think, can be reopened issue https://github.com/OCA/partner-contact/pull/1231?

If module is merged into repo partner-contact, specific data for Ecuador are keep into `l10n_ec_ote`

Reply:
Reopened